src/client/Microsoft.Identity.Client/IConfidentialClientApplication.cs

```diff
@@ -91,8 +91,17 @@ public partial interface IConfidentialClientApplication : IClientApplicationBase
         /// </remarks>
         GetAuthorizationRequestUrlParameterBuilder GetAuthorizationRequestUrl(IEnumerable<string> scopes);
 
-        /// <inheritdoc/>
-        [Obsolete("Use GetAccountAsync in web apps and web APIs, and use a token cache serializer for better security and performance. See https://aka.ms/msal-net-cca-token-cache-serialization.")]
+        /// <summary>
+        /// In confidential client apps use <see cref="IClientApplicationBase.AcquireTokenSilent(IEnumerable{string}, IAccount)"/> instead.
+        /// </summary>
+        [Obsolete("In confidential client apps use AcquireTokenSilent(scopes, account) instead.")]
```

Comment:
Should we not set the "error" flag to true? Is there any scenario where this would work?

Reply:
Static cache/reusing CCA instance? Any test code? Easier to use this than refactoring to GetAccount(userId) since this requires creating an identifier.